### PR TITLE
Remove redundant info about `block_cache_usage` field

### LINKED ISSUE #2231: Improve the output of the block cache usage in INFO command
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Motivation

Currently, all column families are sharing the same block cache since 2.6,
but the info output still displays the block usage per column family which
might confuse users:

```
block_cache_usage[default]:261607392
block_cache_usage[metadata]:261607392
block_cache_usage[zset_score]:261607392
block_cache_usage[stream]:261607392
block_cache_usage[search]:261607392
```

### Solution

Reduce the output format into a single one:

```
block_cache_usage:261607392
```

It will break the old metrics exporter, we should change the kvrocks-exporter as well.

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

### PR DESCRIPTION
**Pull request by jjz921024**
This closes #2231

### CODE REVIEW

File: src/server/server.cc [hunk 1]
```diff
@@ -845,14 +846,16 @@ void Server::GetRocksDBInfo(std::string *info) {
   db->GetAggregatedIntProperty("rocksdb.compaction-pending", &compaction_pending);
   db->GetAggregatedIntProperty("rocksdb.num-live-versions", &num_live_versions);
 
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
```
Comment (git-hulk) [inline src/server/server.cc]:
```suggestion
  // All column families share the same block cache, so it's good to count a single one.
  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
```

**Review by git-hulk — DISMISSED** [review]

File: src/server/server.cc [hunk 2]
```diff
@@ -845,14 +846,17 @@ void Server::GetRocksDBInfo(std::string *info) {
   db->GetAggregatedIntProperty("rocksdb.compaction-pending", &compaction_pending);
   db->GetAggregatedIntProperty("rocksdb.num-live-versions", &num_live_versions);
 
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
```
Comment (PragmaTwice) [inline src/server/server.cc:854]: I think it should be after the `# RocksDB` line, isn‘t it?

**Review by PragmaTwice — CHANGES_REQUESTED** [review]
See above.

File: src/server/server.cc [hunk 3]
```diff
@@ -832,6 +832,7 @@ void Server::GetRocksDBInfo(std::string *info) {
   uint64_t memtable_sizes = 0, cur_memtable_sizes = 0, num_snapshots = 0, num_running_flushes = 0;
   uint64_t num_immutable_tables = 0, memtable_flush_pending = 0, compaction_pending = 0;
   uint64_t num_running_compaction = 0, num_live_versions = 0, num_super_version = 0, num_background_errors = 0;
+  uint64_t block_cache_usage = 0;
```
Comment (PragmaTwice) [inline src/server/server.cc]: This variable declaration should be placed closer to where it is used.

**Review by PragmaTwice — DISMISSED** [review]

File: src/server/server.cc [hunk 4]
```diff
@@ -846,13 +846,18 @@ void Server::GetRocksDBInfo(std::string *info) {
...
+
+  uint64_t block_cache_usage = 0;
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
+
   for (const auto &cf_handle : *storage->GetCFHandles()) {
-    uint64_t estimate_keys = 0, block_cache_usage = 0, block_cache_pinned_usage = 0, index_and_filter_cache_usage = 0;
+    uint64_t estimate_keys = 0, block_cache_pinned_usage = 0, index_and_filter_cache_usage = 0;
```
Comment (mapleFU) [inline src/server/server.cc]: Quick question: would `block_cache_pinned_usage` be shared?

File: src/server/server.cc [hunk 5]
```diff
@@ -846,13 +846,18 @@ void Server::GetRocksDBInfo(std::string *info) {
...
 
   string_stream << "# RocksDB\r\n";
+
+  uint64_t block_cache_usage = 0;
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
+
   for (const auto &cf_handle : *storage->GetCFHandles()) {
```
Comment (jihuayu) [inline src/server/server.cc:856]: Hi, sonar cloud telling us we declaration shadows a local variable is not clear. 
Can you change the variable name?
https://sonarcloud.io/project/issues?open=AY7Xa5ShOhsi8ezKPlra&id=apache_kvrocks

File: src/server/server.cc (hunk 4 above)
Comment (jihuayu) [reply src/server/server.cc]: Can you split each identifier in a dedicated statement
```suggestion
    uint64_t estimate_keys = 0;
    uint64_t block_cache_pinned_usage = 0;
    uint64_t index_and_filter_cache_usage = 0;
```
https://sonarcloud.io/project/issues?open=AY7Xa5ShOhsi8ezKPlrb&id=apache_kvrocks

File: src/server/server.cc [hunk 6]
```diff
@@ -845,14 +846,26 @@ void Server::GetRocksDBInfo(std::string *info) {
   db->GetAggregatedIntProperty("rocksdb.compaction-pending", &compaction_pending);
   db->GetAggregatedIntProperty("rocksdb.num-live-versions", &num_live_versions);
 
+  // All column families share the same block cache, so it's good to count a single one.
+  auto cf_handle = storage->GetCFHandle(engine::kSubkeyColumnFamilyName);
+  db->GetIntProperty(cf_handle, "rocksdb.block-cache-usage", &block_cache_usage);
+  string_stream << "block_cache_usage:" << block_cache_usage << "\r\n";
+
   string_stream << "# RocksDB\r\n";
```
Comment (mapleFU) [inline src/server/server.cc:848]: Why block_cache_usage is before RocksDB? is it being duplicated?

**Review by git-hulk — APPROVED** [review]

**Review by jihuayu — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Pull request merged**